Source/WebCore/editing/Editing.cpp

```diff
@@ -574,6 +574,11 @@ Element* enclosingAnchorElement(const Position& p)
     return nullptr;
 }
 
+Element* enclosingAtomicNode(const Position&)
```

Comment:
You need to name your argument.
See other functions of similar name for an example.